### PR TITLE
add district-tasks

### PR DESCRIPTION
**Pull request by kwladyka**
WIP Waiting for DNS settings

### CODE REVIEW

File: district-tasks/ui/district-tasks.io [hunk 1]
```diff
@@ -0,0 +1,22 @@
+server {
+  listen 80 default_server;
```
Comment (fbielejec) [inline district-tasks/ui/district-tasks.io]: 80, 6300, 5001, 8080, 8545 will be occupied, use different port

File: district-tasks/ui/X0X.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!doctype html>
```
Comment (fbielejec) [inline district-tasks/ui/X0X.html]: reuse existing when building e.g. https://github.com/district0x/x0x

File: district-tasks/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - tasks-net
+    - qa-net
+
+  ui:
+    build: ui
+    container_name: qa_district-tasks-ui
+    depends_on:
+    - server
+    ports:
+    - 80:80
```
Comment (fbielejec) [inline district-tasks/docker-compose.yml]: occupied port

**Review by fbielejec — CHANGES_REQUESTED** [review]

**Comment by kwladyka** [comment]
Added changes

File: district-tasks/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    container_name: qa_district-tasks-server
+    ports:
+    - 6500:6500
+    volumes:
+    - server/config.edn:/app/config.edn
+    environment:
+    - CONFIG=/app/config.edn
+    networks:
+    - tasks-net
+    - qa-net
```
Comment (fbielejec) [inline district-tasks/docker-compose.yml:14]: this network is never created, is it needed?

File: district-tasks/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  server:
+    bulid: server
+    container_name: qa_district-tasks-server
+    ports:
+    - 6500:6500
+    volumes:
+    - server/config.edn:/app/config.edn
```
Comment (fbielejec) [inline district-tasks/docker-compose.yml:9]: why mount a volume just for this config? Just keep the config in container.

File: district-tasks/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+  ui:
+    build: ui
+    container_name: qa_district-tasks-ui
+    depends_on:
+    - server
+    ports:
+    - 81:80
+    volumes:
+    - ui/nginx.conf:/etc/nginx/nginx.conf
```
Comment (fbielejec) [inline district-tasks/docker-compose.yml:24]: no need to mount these volumes, we run separate nginx processes, configs can stay in containers

File: district-tasks/server/smart_contracts.cljs [hunk 7]
```diff
@@ -0,0 +1,9 @@
+(ns district0x-tasks.shared.smart-contracts)
+
```
Comment (fbielejec) [inline district-tasks/server/smart_contracts.cljs:2]: are these addresses part of the repo you build in the containers? If not add the file before building

File: district-tasks/ui/district-tasks.io [hunk 8]
```diff
@@ -0,0 +1,22 @@
+server {
```
Comment (fbielejec) [inline district-tasks/ui/district-tasks.io:1]: use ipfs as proxy, that's how it will run in production.

**Review by fbielejec — CHANGES_REQUESTED** [review]

**Pull request closed without merging**